src/distributions/distribution-factory.ts

```diff
@@ -3,11 +3,14 @@ import { JavaInstallerOptions } from './base-models';
 import { LocalDistribution } from './local/installer';
 import { ZuluDistribution } from './zulu/installer';
 import { AdoptDistribution, AdoptImplementation } from './adopt/installer';
+import { AdoptiumDistribution, AdoptiumImplementation } from './adoptium/installer';
```

Comment:
import { AdoptiumDistribution, AdoptiumImplementation } from './adoptium/installer';

should be:

import { TemurinDistribution, TemurinImplementation } from './temurin/installer';